src/Microsoft.Data.SqlClient/tests/FunctionalTests/GenericConverterTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlTypes;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.Tests
+{
+    public class GenericConverterTests
+    {
+        [Fact]
+        public void ObjectConversionSuccessful()
+        {
+            object test = 0m;
+
```

Comment:
Please remove extra spacing, and could we extend tests to test different types and data ranges?